base_delivery_carrier_files/__manifest__.py

```diff
@@ -57,11 +37,11 @@
              'stock_view.xml',
              'wizard/generate_carrier_files_view.xml',
              'security/ir.model.access.csv'],
-    'demo': ['carrier_file_demo.xml',
-             'carrier_file_demo.yml'],
-    'test': ['test/carrier_file.yml',
-             'test/carrier_file_manual.yml'],
+    'demo': ['carrier_file_demo.xml', 'carrier_file_demo.yml'],
+    'test': [ '../account/test/account_minimal_test.xml',
+              'test/carrier_file.yml',
+              'test/carrier_file_manual.yml'],
```

Comment:
We still should integrate changes from [here](https://github.com/OCA/delivery-carrier/pull/163/files)